index.js

```diff
@@ -1,21 +1,48 @@
-const express = require('express')
+// Set-up
+const express = require('express');
 const exphbs  = require('express-handlebars');
 const mongoose = require('mongoose');
-const app = express()
+const app = express();
+
+// Utility functions
+const utils = require('./controllers/utils');
+
+// Auth items
+const cookieParser = require('cookie-parser');
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+// Bodyparser
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+
+// Auth
+app.use(cookieParser());
+app.use(utils.checkAuth); // Check auth
+// console.log("Post use")
 
 // Resources - Public
-app.use(express.static('public'))
+app.use(express.static('public'));
 
 // Database Models - Mongoose
-mongoose.Promise = global.Promise
-mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost/WEB-3-Project', {useMongoClient: true});
+// NOTE: change the mongodb location as needed, please.
```

Comment:
I think we can remove this, the client doesn't connect to a MongoDB...does it?

Reply:
Traces of mongoose removed. Nice catch.